Source/Csla.Blazor/PropertyInfo.cs

```diff
@@ -28,14 +30,16 @@ public class PropertyInfo : IPropertyInfo, INotifyPropertyChanged
     /// </summary>
     /// <param name="model">Model object</param>
     /// <param name="propertyName">Property name</param>
-    public PropertyInfo(object model, string propertyName)
+    /// <param name="textSeprator">Text Seprator</param>
+    public PropertyInfo(object model, string propertyName, string textSeprator = " ")
```

Comment:
Would this be better as a optional parameter so people who want to use the default can just ignore this new parameter?